curious/graph.py

```diff
@@ -132,19 +132,19 @@ def get_related_objects(instances, filters=None):
 
       table = instances[0]._meta.db_table
       pk_field = instances[0]._meta.pk.column
-      related_table = field.related_field.model._meta.db_table
+      related_table = field.rel.model._meta.db_table
```

Comment:
these three renamings to `rel` don't seem logically consistent with one another; they may in fact be correct, just give them a double-check

Reply:
Checked via print